imap_processing/tests/test_cli.py

```diff
@@ -28,7 +29,14 @@ def mock_instrument_dependencies():
 
 @mock.patch("imap_processing.cli.Mag")
 def test_main(mock_instrument):
-    """Test imap_processing.cli.main()"""
+    """
+    Test imap_processing.cli.main().
+
+    Parameters
+    ----------
+    mock_instrument : str TODO CHeck
```

Comment:
Adding expected types for tests is tricky sometimes because they are mocking other things. I suggest leaving test docstrings for last as they are internal facing rather than external facing.